chia/cmds/wallet_funcs.py

```diff
@@ -234,7 +234,7 @@ async def send(args: dict, wallet_client: WalletRpcClient, fingerprint: int) ->
             return None
 
     print("Transaction not yet submitted to nodes")
-    print(f"Do 'chia wallet get_transaction -f {fingerprint} -tx 0x{tx_id}' to get status")
+    print(f"To get status, use command: 'chia wallet get_transaction -f {fingerprint} -tx 0x{tx_id}'")
```

Comment:
I don't think the `'` add anything now.  Though they aren't in either of your outputs so I'm a bit confused.

```suggestion
    print(f"To get status, use command: chia wallet get_transaction -f {fingerprint} -tx 0x{tx_id}")
```

Reply:
They should have shown in the output.

```pycon
Python 3.8.10 (default, Mar 15 2022, 12:22:08) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> print(f" stuff {1} 'blue {27 + 3}' and some")
 stuff 1 'blue 30' and some
```